test/ModelTraitTest.php

```diff
@@ -27,6 +28,7 @@ class ModelTraitTest extends \PHPUnit_Framework_TestCase {
      *
      * @param $testData
      * @param $expectedProperties
+     * @throws ModelException| \ReflectionException
```

Comment:
For consistency, each Exception should have it's own `@throws` declaration

Reply:
Separated the `@throws`